src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,7 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    private int sequenceNumber;
```

Comment:
You can create a getter function, to allow usage of `sequenceNumber`, since it is a private member.